docs/authoring.md

```diff
@@ -36,23 +36,21 @@ The library you are authoring should specify the following properties in its pro
 ```
 And don't forget to include a `PackageReference` to `Microsoft.Windows.CsWinRT`!
 
-## Generate a NuGet package for the component
-To generate a NuGet package for the component, you can simply right click on the project and select **Pack**. Alternatively, you can add the following property to the library project file to automatically generate a NuGet package on build.
+## Packaging
+To generate a NuGet package for the component, you can simply right click on the project and select **Pack**. Alternatively, you can add the following property to the library project file to automatically generate a NuGet package on build: `GeneratePackageOnBuild`. 
 
-``` csproj
-<PropertyGroup>
-  <GeneratePackageOnBuild>true</GeneratePackageOnBuild> 
-</PropertyGroup>
-```
+To make your component available as a NuGet package, it is important to include the DLLs necessary for C#/WinRT hosting. 
+When you pack your C#/WinRT component the DLLs/WinMD are automatically added to your nupkg, based on a nuspec generated from your project file. 
 
-**If you are going to write your own nuspec, i.e. not rely on automatic packaging** then the CsWinRT target that adds the hosting dlls to your package will not run, and you should make sure your nuspec contains the following ```file``` entries for ```MyAuthoredComponent``` (note: your TargetFramework may vary). 
+**If you are going to write your own nuspec**, then you should make sure your nuspec contains the following ```file``` entries for your component ```MyAuthoredComponent``` (note: your TargetFramework may vary). This is so our targets that supply the DLLs for any consumers of your package work.  
+Similarly, any other dependencies, e.g. `Microsoft.WinUI`, will need to be included in your nuspec as well.
 
 ``` nuspec
 <files>
-  <file src="$(TargetDir)MyAuthoredComponent.dll"        target="lib\native\MyAuthoredComponent.dll" />
+  <file src="$(TargetDir)MyAuthoredComponent.dll"        target="lib\$(TargetFramework)\MyAuthoredComponent.dll" />
   <file src="$(TargetDir)MyAuthoredComponent.winmd"      target="winmd\MyAuthoredComponent.winmd" />

```

Comment:
Is this target directory still accurate, it looks different from below.

Reply:
Our transitive targets expect to find the dlls in a net5.0* directory of their nupkg